zarf/proto/auth.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
```

Comment:
You might consider using 2023 edition and the [opaque API](https://go.dev/blog/protobuf-opaque)

Reply:
You'll need to update the code as well, `protoc` generates different code :)